query/control/controller.go

```diff
@@ -85,7 +85,7 @@ type Config struct {
 	// The context value must be a string or an implementation of the Stringer interface.
 	MetricLabelKeys []string
 
-	ExecutorDependencies execute.Dependencies
+	ExecutorDependencies flux.Dependency
```

Comment:
Should this be an array of dependencies or `flux.Dependencies`?

Reply:
That's not, that is something that implements `flux.Dependency`, the only thing that OSS needs  to know is that it can call `Inject` on it. Indeed, it passes a `influxdb.Dependencies`

Reply:
I was thinking that each dependency would eventually be its own dependency and that `ExecutorDependencies` would just be a slice of all of the dependencies you wanted to inject into the executor. At the moment and possibly for longer, we would group some of the more common ones into a single object, but it might be better to prepare this function signature for something else.

Reply:
Yes, I see your point. I'll do that 👍